### DISCUSSION TITLE
General feedback after first use

### DISCUSSION THREAD

**Discussion by kevinwallimann** [AbsaOSS/generate-release-notes#18] [Ideas]
- Chapter "Issues without Release notes" is not necessary imo. Sometimes you don’t need release notes and if they are really missing you will see it in the release draft, because the release notes are just not there. All of the issues in that chapter will be duplicates, so you always have to remove them.
- I would omit chapters if “No entries detected” -> Unnecessary deletion effort. Deletion is not free, you always have to double-check that you don’t delete something inadvertently
- PRs without linked issue: Would be nice if they could be handled more gracefully, as a first-level citizen like an issue. Sometimes it doesn’t make sense to have an issue for a PRs, e.g. for bug fixes. Also for open-source contributions we can probably not mandate that an issue be created for each PR. 
- Maybe related: Closed PRs (as opposed to merged PRs) can probably also be omitted. Arguably they are rare, but I can't think of a scenario where a closed PR should still be in the release notes. But just my point of view.

**Comment by miroslavpojer** [comment]
```
Chapter "Issues without Release notes" is not necessary imo. Sometimes you don’t need release notes and if they are really missing you will see it in the release draft, because the release notes are just not there. All of the issues in that chapter will be duplicates, so you always have to remove them.
```

This chapter still could provide useful information when the presence of Release notes is required by project rules. 
Here is a planned [issue](https://github.com/AbsaOSS/generate-release-notes/issues/15) to add skip logic into action. With this improvement, the team can mark Issues without Release notes comments by intention.

**Comment by miroslavpojer** [comment]
```
I would omit chapters if “No entries detected” -> Unnecessary deletion effort. Deletion is not free, you always have to double-check that you don’t delete something inadvertently
```
I have created a new feature [issue](https://github.com/AbsaOSS/generate-release-notes/issues/23) to address this idea.

**Comment by miroslavpojer** [comment]
```
Maybe related: Closed PRs (as opposed to merged PRs) can probably also be omitted. Arguably they are rare, but I can't think of a scenario where a closed PR should still be in the release notes. But just my point of view.
```
You have identified the not covered area. Thanks

The main idea of this GH action is to provide the "Final check". It is the default setting too.
See [issue](https://github.com/AbsaOSS/generate-release-notes/issues/24)

**Comment by miroslavpojer** [comment]
```
PRs without linked issue: Would be nice if they could be handled more gracefully, as a first-level citizen like an issue. Sometimes it doesn’t make sense to have an issue for a PRs, e.g. for bug fixes. Also for open-source contributions we can probably not mandate that an issue be created for each PR.
```

Do you have some suggestions for change here?

**Update:**
After call I have created new [issue](https://github.com/AbsaOSS/generate-release-notes/issues/26)

**Comment by miroslavpojer** [comment]
Issue solved.
